x11-libs/libxfont/libxfont-1.5.4.recipe

```diff
@@ -1,31 +1,41 @@
-SUMMARY="X Font rasterisation library"
-DESCRIPTION="libXfont provides various services for X servers, most notably \
-font selection and rasterisation (through external libraries such as freetype)."
-HOMEPAGE="http://xorg.freedesktop.org/releases/individual/lib/"
-COPYRIGHT="1994-1996 Silicon Graphics Computer Systems, Inc."
+SUMMARY="X Resize, Rotate and Reflection extension library"
```

Comment:
This summary comes from libxrander :)

Reply:
Fixed, travis has some problems today it seems ...